freqtrade/constants.py

```diff
@@ -159,6 +159,7 @@
         'ignore_buying_expired_candle_after': {'type': 'number'},
         'trading_mode': {'type': 'string', 'enum': TRADING_MODES},
         'margin_mode': {'type': 'string', 'enum': MARGIN_MODES},
+        'convert_df_to_32bit': {'type': 'number', 'default': False},
```

Comment:
what do you think about a more ... generic name for the parameter?

it's true that we're converting to 32bit for now - but we could just as well convert to 16bit ints (for example) in the future - so maybe a name like `reduce_df_footprint` might keep more possibilities open without explicitly stating the final size.

Reply:
Sure go for it!